src/applications/vaos/services/appointment/index.js

```diff
@@ -140,3 +186,148 @@ export function getVAAppointmentLocationId(appointment) {
 
   return null;
 }
+
+/**
+ * Returns the location name of a VA appointment
+ *
+ * @export
+ * @param {Object} appointment A FHIR appointment resource
+ * @returns The location name where the VA appointment is located
+ */
+export function getVAAppointmentLocationName(appointment) {
+  return appointment.participant?.find(p =>
+    p.actor.reference?.startsWith('Location'),
+  )?.actor?.display;
+}
+
+/**
+ * Returns the patient phone number of a VA appointment
+ *
+ * @export
+ * @param {Object} appointment A FHIR appointment resource
+ * @returns The patient phone number where the VA appointment is located
+ */
+export function getPatientPhone(appointment) {
+  return appointment?.participant
+    .find(p => p?.actor?.reference.includes('Patient'))
+    ?.actor?.telecom?.find(t => t.system === 'phone')?.value;
+}
+
+/**
+ * Returns the patient email of a VA appointment
+ *
+ * @export
+ * @param {Object} appointment A FHIR appointment resource
+ * @returns The patient email where the VA appointment is located
+ */
+export function getPatientEmail(appointment) {
+  return appointment.participant
+    .find(p => p?.actor?.reference.includes('Patient'))
+    ?.actor?.telecom?.find(t => t?.system === 'email')?.value;
+}
+
+/**
+ * Filter method for future confirmed appointments
+ * @param {Object} appt A FHIR appointment resource
+ */
+export function filterFutureConfirmedAppointments(appt) {
+  const apptDateTime = moment(appt.start);
+
+  return (
+    !appt.vaos.isPastAppointment &&
+    !FUTURE_APPOINTMENTS_HIDDEN_SET.has(appt.description) &&
+    apptDateTime.isValid() &&
+    apptDateTime.isBefore(moment().add(13, 'months'))
+  );
+}
+
+/**
+ * Filter method for past appointments
+ * @param {Object} appt A FHIR appointment resource
+ */
+export function filterPastAppointments(appt, startDate, endDate) {
+  const apptDateTime = moment(appt.start);
+
+  if (
+    appt.vaos.appointmentType === APPOINTMENT_TYPES.vaAppointment &&
+    PAST_APPOINTMENTS_HIDDEN_SET.has(appt.description)
+  ) {
+    return false;
+  }
+
+  return (
+    apptDateTime.isValid() &&
+    apptDateTime.isAfter(startDate) &&
+    apptDateTime.isBefore(endDate)
+  );
+}
+
+/**
+ * Filter method for past appointment requests
+ * @param {Object} request A FHIR appointment resource
+ */
+export function filterRequests(request) {
+  const now = moment();
+  const hasValidDate = request.requestedPeriod.some(period => {
+    const momentStart = moment(period.start);
+    const momentEnd = moment(period.end);
+    return (
+      momentStart.isValid() && momentStart.isAfter(now) && momentEnd.isValid()
+    );
+  });
+
+  return (
+    !request.vaos.isPastAppointment &&
```

Comment:
`vaos.isPastAppointment` was removed in the transformPendingAppointments function. Should this check be removed?